.github/workflows/run-tests.yml

```diff
@@ -0,0 +1,23 @@
+name: Run Unit tests
+run-name: test-runner
+on: [push]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.6.15]
```

Comment:
This seems like an oddly specific version of Python, could you please test on 3.8, 3.9, 3.10?